install.sh

```diff
@@ -96,7 +96,7 @@ echo "================ Installing apache-maven $APACHE_MAVEN ================="
 wget http://www-eu.apache.org/dist/maven/maven-3/$APACHE_MAVEN/binaries/apache-maven-$APACHE_MAVEN-bin.tar.gz
 tar xzf apache-maven-$APACHE_MAVEN-bin.tar.gz -C /usr/local && rm -f apache-maven-$APACHE_MAVEN-bin.tar.gz
 ln -fs /usr/local/apache-maven-$APACHE_MAVEN/bin/mvn /usr/bin
-echo 'export PATH=$PATH:/usr/local/apache-maven-$APACHE_MAVEN/bin' >> /etc/drydock/.env
+echo 'M2_HOME=/usr/local/apache-maven-$APACHE_MAVEN/' >> /etc/drydock/.env
```

Comment:
This is wrong. We both the path and M2_HOME.

Reply:
ok i will update